### ISSUE TITLE
Hovering over a selected comment causes a ghc-mod crash

### ISSUE THREAD

**Issue by lexi-lambda** [atom-haskell/haskell-ghc-mod#106]
Hovering over a comment normally works, but if I *select a comment and then hover over that*, then I get a loud error message.

![bug](https://cloud.githubusercontent.com/assets/759911/12832119/03a2fd98-cb4d-11e5-8044-dfb9ad08832a.gif)

**Atom Version**: 1.4.3
**System**: Mac OS X 10.11.3
**ghc-mod version**: 5.5.0
**stack resolver**: 5.0
**haskell-ghc-mod version**: 1.6.5

### Log / Stack Trace

```
Haskell-ghc-mod: ghc-mod interactive command info failed with error ghc-modi crashed
```
```
caps: {"version":[5,5,0,0],"legacyInteractive":true,"fileMap":true,"rootExec":false,"quoteArgs":true}
URI: /Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs
Args: -- This is a comment!
message: NG Invalid option `-- This is a comment!'\n\nUsage:  info FILE SYMBOL\n  Look up an identifier in the context of FILE (like ghci's `:info')
log:
-19.733s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-19.733s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-19.733s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-19.733s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-19.732s: Running ghc-modi command info,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,Bin
-19.588s: Got response from ghc-modi:
type Bin = (Integer, Int)  	-- Defined at src/Homework/Week03/Assignment.hs:28:1
OK
-19.588s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-16.275s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-16.275s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-16.275s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-16.274s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-16.274s: Running ghc-modi command info,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,
-16.15s: ghc-modi said: EXCEPTION: info:
-16.149s: ghc-modi said:                parse error (possibly incorrect indentation or mismatched brackets)
-16.149s: Got response from ghc-modi:
Cannot show info
OK
-16.149s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-16.149s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-16.149s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-16.149s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-16.149s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-16.148s: Running ghc-modi command type,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,19,3
-16.025s: Got response from ghc-modi:
OK
-16.025s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-15.339s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-15.339s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-15.339s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-15.338s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-15.338s: Running ghc-modi command info,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,
-15.217s: ghc-modi said: EXCEPTION: info:
-15.217s: ghc-modi said:                parse error (possibly incorrect indentation or mismatched brackets)
-15.217s: Got response from ghc-modi:
Cannot show info
OK
-15.217s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-15.217s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-15.217s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-15.217s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-15.217s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-15.216s: Running ghc-modi command type,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,19,4
-15.094s: Got response from ghc-modi:
OK
-15.094s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-9.455s: Data.List.Split created
-9.455s: Data.List.Split updating
-9.455s: running ghc-mod browse,-d,Data.List.Split with options.cwd = /Users/jaking/gits/cj/cis-194-winter-2016,options.env = [object Object],options.encoding = utf-8
-9.443s: Data.List.Split moduleName mismatch: Homework.Week04.Assignment != Data.List.Split
-9.443s: Data.List.Split moduleName mismatch: Homework.Week03.Assignment != Data.List.Split
-9.442s: Prelude created
-9.442s: Prelude updating
-9.442s: running ghc-mod browse,-d,Prelude with options.cwd = /Users/jaking/gits/cj/cis-194-winter-2016,options.env = [object Object],options.encoding = utf-8
-9.438s: Prelude moduleName mismatch: Homework.Week04.Assignment != Prelude
-9.438s: Prelude moduleName mismatch: Homework.Week03.Assignment != Prelude
-8.902s: ghc-mod ended with code 0
-8.901s: Data.List.Split updated
-8.843s: ghc-mod ended with code 0
-8.841s: Prelude updated
-4.552s: running ghc-mod check,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs with options.cwd = /Users/jaking/gits/cj/cis-194-winter-2016,options.env = [object Object],options.encoding = utf-8
-4.548s: running ghc-mod lint,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs with options.cwd = /Users/jaking/gits/cj/cis-194-winter-2016,options.env = [object Object],options.encoding = utf-8
-3.937s: ghc-mod ended with code 0
-3.813s: ghc-mod ended with code 0
-2.935s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-2.935s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-2.935s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-2.934s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-2.933s: Running ghc-modi command info,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,comment
-2.802s: ghc-modi said: EXCEPTION: info:     Not in scope: ‘comment’
-2.802s: Got response from ghc-modi:
Cannot show info
OK
-2.802s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-2.802s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-2.802s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-2.802s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-2.802s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-2.802s: Running ghc-modi command type,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,9,17
-2.677s: Got response from ghc-modi:
OK
-2.677s: Ended interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-0.013s: Trying to run ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-0.013s: Checking for ghc-modi in /Users/jaking/gits/cj/cis-194-winter-2016
-0.013s: Found running ghc-modi instance for /Users/jaking/gits/cj/cis-194-winter-2016
-0.012s: Started interactive action block in /Users/jaking/gits/cj/cis-194-winter-2016
-0.012s: Running ghc-modi command info,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs,-- This is a comment!
-0.01s: Got response from ghc-modi:
NG Invalid option `-- This is a comment!'\n\nUsage:  info FILE SYMBOL\n  Look up an identifier in the context of FILE (like ghci's `:info')
-0.001s: ghc-modi for /Users/jaking/gits/cj/cis-194-winter-2016 ended with 1
-0.001s: ghc-modi for /Users/jaking/gits/cj/cis-194-winter-2016 ended with 1
-0.001s: Running ghc-modi command unmap-file,/Users/jaking/gits/cj/cis-194-winter-2016/src/Homework/Week03/Assignment.hs
-0.001s: ghc-modi crashed: NG Invalid option `-- This is a comment!'\n\nUsage:  info FILE SYMBOL\n  Look up an identifier in the context of FILE (like ghci's `:info')
```
```
ghc-modi crashed: NG Invalid option `-- This is a comment!'\n\nUsage:  info FILE SYMBOL\n  Look up an identifier in the context of FILE (like ghci's `:info')
    at module.exports.Util.mkError (/Users/jaking/.atom/packages/haskell-ghc-mod/lib/util.coffee:154:15)
    at ChildProcess.exitCallback (/Users/jaking/.atom/packages/haskell-ghc-mod/lib/ghc-mod/interactive-process.coffee:76:20)
    at emitTwo (events.js:92:20)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```

**Comment by lierdakil** [comment]
Should be fixed by v1.6.6.

**Comment by lexi-lambda** [comment]
Aha, glancing at the fix, I probably should have realized what was going on! Thanks a lot, though, haha, I appreciate the quick response. This is a great tool.
